src/main/java/de/tum/in/www1/artemis/service/connectors/bamboo/BambooBuildPlanService.java

```diff
@@ -444,12 +445,12 @@ private PlanPermissions generatePlanPermissions(String bambooProjectKey, String
         return new PlanPermissions(new PlanIdentifier(bambooProjectKey, bambooPlanKey)).permissions(permissions);
     }
 
-    private List<Task<?, ?>> readScriptTasksFromTemplate(final ProgrammingLanguage programmingLanguage, String subDirectory, final boolean sequentialBuildRuns,
```

Comment:
JavaDoc missing

Reply:
I actually have no clue, what this method is doing. There may be changed lines due to the changed type of Task.